### PR TITLE
Fixes size of zoow buttons

### PR DESCRIPTION
**Pull request by cquiroz**
Fix small regression to the zoom buttons' size

### CODE REVIEW

File: explore/src/main/scala/explore/targeteditor/AladinContainer.scala [hunk 1]
```diff
@@ -282,13 +281,15 @@ object AladinContainer {
                   ExploreStyles.AladinZoomControl,
                   Button(
                     size = Small,
+                    icon = true,
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targeteditor/AladinContainer.scala:284]: This is the core of the fix

File: explore/src/main/scala/explore/targeteditor/AladinContainer.scala [hunk 2]
```diff
@@ -43,26 +43,22 @@ final case class AladinContainer(
...
+  type Props = AladinContainer
 
   // This is used for screen coordinates, thus it doesn't need a lot of precission
-  implicit val doubleReuse: Reusability[Double]                  = Reusability.double(1.0)
-  implicit val selectedGSReuse: Reusability[Option[AgsAnalysis]] =
-    Reusability.by(_.map(_.target.id))
-  implicit val analysisReuse: Reusability[List[AgsAnalysis]]     = Reusability.by(_.length)
+  given Reusability[Double]              = Reusability.double(1.0)
+  given Reusability[Option[AgsAnalysis]] = Reusability.by(_.map(_.target.id))
+  given Reusability[List[AgsAnalysis]]   = Reusability.by(_.length)
```
Comment (toddburnside) [inline explore/src/main/scala/explore/targeteditor/AladinContainer.scala:61]: It's so nice not to have to come up with names that are never used. 😄

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/targeteditor/AladinContainer.scala (hunk 2 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/targeteditor/AladinContainer.scala:61]: Yes, it s

**Pull request merged**